org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/ast/Command.java

```diff
@@ -39,6 +39,12 @@
  * <b>Invariant:</b> maxseq >= -1
  * <p>
  * <b>Invariant:</b> maxstring >= -1
+ * <p>
+ * <b>Invariant:</b> mintime == -1 or >= 1
+ * <p>
+ * <b>Invariant:</b> maxtime == -1 or >= 1
+ *
+ * @modified Nuno Macedo // [HASLab] electrum-temporal
```

Comment:
adds two special scopes to commands, the minimum and maximum trace prefix lengths to be considered during analysis; when they are omitted, defaults to -1